docs/data/material/migration/upgrade-to-v6/upgrade-to-v6.md

```diff
@@ -233,7 +238,69 @@ As the `ListItem` no longer supports these props, the class names related to the
 + listItemButtonClasses.selected
 ```
 
-#### Stabilized API
+### Loading Button
+
+In v6, the `children` prop passed to the Loading Button component is now wrapped in a `<span>` tag to avoid [issues](https://github.com/mui/material-ui/issues/27853) when using tools to translate websites.
+
+### Rating
+
+Previously, due to a bug, the `aria-label` attribute was "null Stars" when no value was set in the Rating component.
+This is fixed in v6, with the `aria-label` attribute being "0 Stars" when no value is set.
+
+### useMediaQuery types
+
+The following deprecated types are removed in v6:
+
+- `MuiMediaQueryList`: use `MediaQueryList` (from lib.dom.d.ts) instead.
+- `MuiMediaQueryListEvent`: use `MediaQueryListEvent` (from lib.dom.d.ts) instead.
+- `MuiMediaQueryListListener`: use `(event: MediaQueryListEvent) => void` instead.
+
+## Breaking changes affecting testing
+
+### Ripple effect
+
+The ripple effect's performance has been improved in v6.
+Because of this, you might need to update tests involving components with the ripple effect.
+If you are using `fireEvent` from `@testing-library/react` to simulate user interactions, you will need to wrap these inside `act` and `await` to avoid React warnings:
+
+```diff
+- fireEvent.click(button);
++ await act(async () => fireEvent.mouseDown(button));
+```
+
+The components affected by this change are:
+
+- All buttons
+- Checkbox
+- Chip
+- Radio Group
+- Switch
+- Tabs
+
+## Stabilized APIs
+
+### CssVarsProvider and extendTheme
+
+The `CssVarsProvider` and `extendTheme` APIs are now stable.
+If you're already using them in v5 you can now drop the experimental prefix:
```

Comment:
Should we point users to `ThemeProvider` instead?

Reply:
Yes, although it is currently handled in https://github.com/mui/material-ui/pull/43130/files#diff-c55db4ddcf075fc8361b2c85811c886426a5eb61f6bcb1b70046009b2517ffc9R350, except for changing the proposed import which we should also do.

I'll coordinate with Jun so the final version points to `ThemeProvider`

Reply:
Done in https://github.com/mui/material-ui/pull/43130/commits/c20365bf2ff4674022c9bff07d4125af13335e9a